src/main/java/pl/piekoszek/gorskimatches/facebook/FacebookController.java

```diff
@@ -30,10 +34,8 @@ public ResponseEntity<String> get(@RequestParam(name = "hub.verify_token") Strin
 
     @PostMapping
     public void post(@RequestBody FacebookHookRequest request) {
```

Comment:
BTW, in FacebookHookRequest `object` field will always be null
If field has no Getter and is not public, will not be set during JSON deserialization

Anyway it's not used so can be removed